deploy/examples/external/dashboard-external-http.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
```

Comment:
@travisn we never document how to use the dashboard for external cluster, do you think there a need for adding this to the document